packages/s3-presigned-post/src/createPresignedPost.integ.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { NoSuchKey, S3 } from "@aws-sdk/client-s3";
+const FormData = require("form-data");
+import { createReadStream, rmSync, writeFileSync } from "fs";
+import { join } from "path";
+
+import { createPresignedPost } from "./createPresignedPost";
+
+describe(createPresignedPost.name, () => {
+  const Bucket = "aws-sdk-js-integration-test-s3-presigned-post";
+  const Key = "temp-file.txt";
+  const contents = "Hello, world!";
+  const fileLocation = join(__dirname, Key);
+  const client = new S3({ region: "us-east-1", endpoint: "https://s3-fips.dualstack.us-east-1.amazonaws.com" });
+
+  it("should allow custom endpoints to be modified by endpoint resolution options", async () => {
+    {
+      const client = new S3({
+        region: "us-east-1",
+        forcePathStyle: true,
+        endpoint: "https://s3-fips.dualstack.us-east-1.amazonaws.com",
+      });
+      const { url } = await createPresignedPost(client, {
+        Bucket,
+        Key,
+      });
```

Comment:
nit: params to be on single line
```suggestion
      const { url } = await createPresignedPost(client, { Bucket, Key });
```

ditto at other places.